src/precice/SolverInterface.cpp

```diff
@@ -72,296 +80,296 @@ bool SolverInterface::requiresWritingCheckpoint()
   return _impl->requiresWritingCheckpoint();
 }
 
-bool SolverInterface::hasMesh(std::string_view meshName) const
+bool SolverInterface::hasMesh(::precice::string_view meshName) const
 {
-  return _impl->hasMesh(meshName);
+  return _impl->hasMesh(toSV(toSV(meshName)));
```

Comment:
```suggestion
  return _impl->hasMesh(toSV(meshName));
```
Seems like there are a couple of instances where this occurs twice. Please check

Reply:
Done, thanks for pointing it out.